File: LeetCode Solutions/May 2020 LeetCoding Challenge/Minimum Moves to Equal Array Elements II.py
class Solution:
    def minMoves2(self, nums: List[int]) -> int:
        nums.sort()
        minimumNumberOfMoves, median = 0, nums[len(nums) // 2]
        for num in nums:
            minimumNumberOfMoves += abs(median - num)
        return minimumNumberOfMoves

File: LeetCode Solutions/May 2020 LeetCoding Challenge/Minimum Moves to Equal Array Elements II.cpp
class Solution {
public:
    int minMoves2(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int minimumNumberOfMoves = 0, median = median = nums[~~(nums.size() / 2)];
        for(int num : nums){
            minimumNumberOfMoves += abs(median - num);
        }
        return minimumNumberOfMoves;
    }
};